CHANGELOG.md

```diff
@@ -22,6 +22,10 @@
 
       https://github.com/solidusio/solidus/pull/1461
 
+    * Removed `#perform!` callback from refund model, it must be called manually
```

Comment:
I'd suggest wording this more strongly somehow.  Refunds really should always be performed via `create_and_perform!` unless you are taking care to delete the refund if it fails (via a transaction or otherwise).  And if you're importing data you won't be "performing" the refund.  Right?